src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -68,6 +68,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- Exclude GAC, registry, output directory from search paths. -->
     <AssemblySearchPaths>{CandidateAssemblyFiles};{HintPathFromItem};{TargetFrameworkDirectory};{RawFileName}</AssemblySearchPaths>
     <DesignTimeAssemblySearchPaths>$(AssemblySearchPaths)</DesignTimeAssemblySearchPaths>
+
+    <AllowUnsafeBlocks Condition="'$(AllowUnsafeBlocks)'==''">false</AllowUnsafeBlocks>
+    <TreatWarningsAsErrors Condition="'$(TreatWarningsAsErrors)'==''">false</TreatWarningsAsErrors>
+    <Prefer32Bit Condition="'$(Prefer32Bit)'==''">false</Prefer32Bit>
+    <SignAssembly Condition="'$(SignAssembly)'==''">false</SignAssembly>
+    <DelaySign Condition="'$(DelaySign)'!=''">false</DelaySign>
```

Comment:
Good catch, that's getting wiped away (details on the thread) so I missed that. I'll send out another PR with an update.